internal/handler/company_handler.go

```
@@ -0,0 +1,55 @@
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(res.ErrorResponse{
+			Message: "error",
+			Error:   "Invalid request format",
+		})
+		return
+	}
+
+	err = h.createUsecase.CreateCompany(req)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(res.ErrorResponse{
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)